create3_lidar/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+
+cmake_minimum_required(VERSION 3.5)
+project(create3_lidar)
+
+# Default to C++14
```

Comment:
minor: this package contains only python files, so it would be better to have a python-based setup rather than CMake file.
if we keep CMake, you can remove lines 5 -> 12